Source/WebCore/svg/SVGGraphicsElement.cpp

```diff
@@ -97,7 +97,7 @@ AffineTransform SVGGraphicsElement::animatedLocalTransform() const
     AffineTransform matrix;
 
     CheckedPtr renderer = this->renderer();
-    auto* style = renderer ? &renderer->style() : nullptr;
+    CheckedPtr style = renderer ? &renderer->style() : nullptr;
```

Comment:
Isn't `style->hasTransform()` trivial? cc @rniwa

Reply:
The issue is that this code block contains function calls to applyTransform, computeTransformOrigin, etc...

Reply:
Oh, I should have expanded the diff, `style` is used quite a bit more below indeed 👍🏻